internal/sequencer/types.go

```diff
@@ -254,7 +254,7 @@ func (tr *TransferResponse) JSON() ([]byte, error) {
 }
 
 func (tr *TransferResponse) TableHeader() []string {
-	return []string{"From", "To", "Amount", "Nonce", "TxHash"}
+	return []string{"From", "To", "Nonce", "Amount", "TxHash"}
```

Comment:
Ah I caused this when I updated the order but not the rows. Let's match the order of the struct fields and instead update `TableRows` below